optimizely/optimizely.py

```diff
@@ -487,6 +492,10 @@ def get_feature_variable_double(self, feature_key, variable_key, user_id, attrib
       - Mismatch with type of variable.
     """
 
+    if not self.is_valid:
+      self.logger.error(enums.Errors.INVALID_DATAFILE.format('get_feature_variable_double'))
```

Comment:
'get_feature_variable_boolean' is not version number. It should be something else or config.version?

Reply:
This is other error. It's been used throughout all the APIs. 
This will print "Invalid Datafile. Failing get_feature_variable_boolean"